### ISSUE TITLE
Negative Frequencies

### ISSUE THREAD

**Issue by faridf** [rehnd/MoS2-band-structure#1]
Hi there!

Thanks for providing all these detailed calculations. Do you happen to know why there are 3 negative frequencies at the gamma point?
https://github.com/rehnd/MoS2-band-structure/blob/master/vasp/phonon/2-dfpt/band.yaml

**Comment by rehnd** [comment]
Yes this is a small cell, chosen to run quickly. Testing convergence with respect to cell size and k mesh is needed and should resolve the imaginary frequenciesOn May 24, 2023, at 8:59 PM, faridf ***@***.***> wrote:﻿
Hi there!
Thanks for providing all these detailed calculations. Do you happen to know why there are 3 negative frequencies at the gamma point?
https://github.com/rehnd/MoS2-band-structure/blob/master/vasp/phonon/2-dfpt/band.yaml

**Comment by faridf** [comment]
Thanks. Then my follow-up question would be, how is that your entropy coming out correctly at your website?
https://rehnd.github.io/tutorials/vasp/phonons

I did try to increase the cell size to 96 atoms but that still did not help with the negative frequencies. I used the Tags you provided in your INCAR. I have done this sort of runs before and I never encountered such thing before but for MoS2, I am seeing negative frequencies which is curious.
